.github/workflows/tests.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
     container: node:20.7-bullseye
     services:
       postgres:
-        image: postgres:16.1-alpine
+        image: pgvector/pgvector:pg16
```

Comment:
Regarding our conversation @sidravi1, to make pgvector work, we need to run the pgvector image AND to create the extension.